src/OpenSage.Game/Logic/Object/ObjectStatus.cs

```diff
@@ -66,6 +66,15 @@ public enum ObjectStatus
         [IniEnum("IS_BRAKING")]
         IsBraking,
 
+        [IniEnum("IS_ATTACKING")]
+        IsAttacking,
+
+        [IniEnum("IS_FIRING_WEAPON")]
+        IsFiringWeapon,
+
+        [IniEnum("IS_AIMING_WEAPON")]
+        IsAimingWeapon,
+
```

Comment:
Unrelated to this PR, I just happened to notice these while loading ZH .sav files.